package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "bytezone",
   "version": "0.1.0",
-  "description": "",
-  "main": "index.js",
+  "description": "Static website served via Github pages",
   "type": "module",
   "scripts": {
-    "build": "npx @11ty/eleventy",
-    "dev": "npx @11ty/eleventy --serve",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "eleventy",
```

Comment:
Are these commands run with `npx`? What is the difference with just "eleventy"?

Reply:
`npx` is a duplicative tool in this case. Nothing about the behavior changes. Just using `npx` add a sub-process, and additional package.json lookup.